vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -275,11 +276,13 @@ def fit(self,
         if time_limit is None and num_trials is None:
             raise ValueError('`time_limit` and `num_trials` can not be `None` at the same time, '
                              'otherwise the training will not be terminated gracefully.')
-        config={'log_dir': self._log_dir,
-                'num_trials': 99999 if num_trials is None else max(1, num_trials),
-                'time_limits': 2147483647 if time_limit is None else max(1, time_limit),
-                'search_strategy': search_strategy,
-                }
+        config = {'log_dir': self._log_dir,
+                  'num_trials': 99999 if num_trials is None else max(1, num_trials),
+                  'time_limits': 2147483647 if time_limit is None else max(1, time_limit),
+                  'searcher': searcher,
+                  'search_strategy': searcher,  # needed for gluon-cv
```

Comment:
maybe add TODO that this can be fixed eventually?

Reply:
added TODO + created issue in gluon-cv: https://github.com/dmlc/gluon-cv/issues/1633